Sources/JWT/Signers/RSAKey+JWK

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import CTLS
+
+public extension RSAKey {
+
+    public init(n: String, e: String, d: String? = nil) throws {
+
+        func parseBignum(_ s: String) -> UnsafeMutablePointer<BIGNUM> {
+            return s.makeBytes().base64URLDecoded.withUnsafeBufferPointer { p in
+                return BN_bin2bn(p.baseAddress, Int32(p.count), nil)
+            }
+        }
+
+        let rsa = RSA_new()!
```

Comment:
Prefer `guard let rsa = ...` over force unwrapping. You can throw `JWTError.createKey`.